docs/pages/blog/aggregation-functions.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Aggregate data like in Excel, but easier!
+description: Introducing aggregation functions and summary rows in Data grid Premium.
+date: 2022-08-01T00:00:00.000Z
+authors: ['josefreitas', 'flaviendelangle', 'cherniavskii']
+tags: ['MUI X', 'News']
+card: false
+---
+
+If you ever had a lot of data on a grid, you probably know that the ability to set different perspectives on data is essential to help users extract information. Typically, one would use at least filtering or sorting, but that barely scratches the surface of what one could do.
+
+The Data grid Premium is all about enabling and empowering advanced use cases for data analysis.
+And in that spirit, we’re overjoyed to announce that starting from [v5.15.0](https://github.com/mui/mui-x/releases/tag/v5.15.0), you can now use [aggregation functions](/x/react-data-grid/aggregation/), and with a couple of clicks, extract information like **sum**, **average**, **count**, and others.
+
+## Wait, what is an aggregation function?
```

Comment:
This article uses two different styles of headers: some are styled as the reader asking questions, while others simply describe the respective section. I would suggest writing them all in the latter style for consistency.

```suggestion
## What are aggregation functions?
```

Reply:
I understand the argument of style consistency, but I think it's harmless in this case, and both headers using this focus on genuine questions users are likely to make.